### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by kdgromova**
Added profile page where user can update their information, change passwords, etc.

### CODE REVIEW

File: MedicationScheduler/AuthApp/templates/profile.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                      {{ password_change_form.as_p }}
+                      <button type="submit" class="btn btn-primary" name="change_password">Save changes</button>
+                    </form>
+              </div>
+            </div>
+            <div class="tab-pane fade" id="account-info">
+              <div class="card-body">
+                <form method="post">
+                      {% csrf_token %}
+                      {{ update_profile_form.as_p }}
```
Comment (bew5294) [inline MedicationScheduler/AuthApp/templates/profile.html:48]: I can't leave a field empty in the update profile form.

File: MedicationScheduler/AuthApp/views.py [hunk 2]
```diff
@@ -61,3 +60,27 @@ def login_view(request):
...
+            password_change_form.save()
+            return redirect('Auth:login')
+    elif request.method == 'POST' and 'user_update' in request.POST:
+        user_update_form = UpdateUserForm(request.POST, instance=request.user)
+        if user_update_form.is_valid():
+            user_update_form.save()
+    elif request.method == 'POST' and 'update_profile_info' in request.POST:
+        update_profile_form = UpdateUserProfile(request.POST, instance=request.user)
+        if update_profile_form.is_valid():
+            update_profile_form.save()
```
Comment (bew5294) [inline MedicationScheduler/AuthApp/views.py:81]: User profile information is not being saved into the database on my end.

**Review by bew5294 — CHANGES_REQUESTED** [review]
I noted a few problems I encountered when trying to update a user profile.

File: MedicationScheduler/AuthApp/templates/profile.html (hunk 1 above)
Comment (kdgromova) [reply MedicationScheduler/AuthApp/templates/profile.html:48]: @bew5294 done

File: MedicationScheduler/AuthApp/views.py (hunk 2 above)
Comment (kdgromova) [reply MedicationScheduler/AuthApp/views.py:81]: @bew5294 good catch. thank you! It should work now

File: MedicationScheduler/AuthApp/templates/profile.html (hunk 1 above)
Comment (bew5294) [reply MedicationScheduler/AuthApp/templates/profile.html:48]: @kdgromova looks like everything is working except when I leave the birthday field empty I get the following error. I'm not sure why:
image

Comment (kdgromova) [reply MedicationScheduler/AuthApp/templates/profile.html:48]: @bew5294 that's very strange. It works on my end.

Comment (kdgromova) [reply MedicationScheduler/AuthApp/templates/profile.html:48]: oh, now I got this problem.

Comment (kdgromova) [reply MedicationScheduler/AuthApp/templates/profile.html:48]: @bew5294 I fixed it

**Review by bew5294 — APPROVED** [review]
looks good

**Pull request merged**